config/initializers/quick_classification_bug_fix.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+# NOTE: This issue only seems to present in development, and not consistently.
+# Compact the available work types to remove `nil` and prevent an `Nil location provided. Can't build URI.` error
+# when not all options are selected, thrown from `SelectTypePresenter#switch_to_new_work_path`
+Hyrax::QuickClassificationQuery.class_eval do
+  def normalized_model_names
+    models.map { |name| concern_name_normalizer.call(name) if Site.first.available_works.include?(name) }.compact
```

Comment:
Do we know if this bug fix is still required, and do we have tests which relate to it?